### DISCUSSION TITLE
Importing Procreate brushes

### DISCUSSION THREAD

**Discussion by aliciedraws** [keiwando/toonsquid#337] [Q&A]
I can't seem to find a way to import Procreate Brushes. I've checked the official Youtube page and searched through the Q&A posts. Is this a feature in development currently, or is it already possible? If so, does anyone mind briefly guiding me through how? thank you!

**Comment by trinhpxl** [comment]
not currently possible, see https://github.com/keiwando/toonsquid/discussions/156#discussioncomment-5552156
